README.md

```diff
@@ -1 +1,39 @@
-##Network Service
+# Network Service 
+
+![macOS](https://img.shields.io/badge/macOS-13.3+-blue) ![Xcode](https://img.shields.io/badge/Xcode-14.3+-red)  ![Xcode](https://img.shields.io/badge/Xcode-14.3+-red)  ![Swift](https://img.shields.io/badge/Swift-5.8-orange) ![iOS](https://img.shields.io/badge/iOS-15%2B-lightgrey)
+
+The repo contains a Swift Package with the implementation of the HTTP network service for the iOS platform.
+
+⚠️ The Network service is under active development.
+
+## Table of contents
+
+* [Installation](#installation)
+* [Authors](#authors)
+
+## Installation
+
+**Pre-requisites**: _installed Xcode with the version specified under the [current dev environment](#current-dev-environment)._
+
+To install Network Service package via Xcode
+
+1. Go to `File` -> `Add Packages`
+1. Then search for `https://github.com/ios-course/simple-network-service.git`
+1. Click `Add Package`
+1. Leave default settings
```

Comment:
I've tried to follow you instruction and get the following warning:

<img width="1531" alt="Screenshot 2023-06-19 at 15 28 19" src="https://github.com/ios-course/simple-network-service/assets/6073510/6185474f-71e6-4ef0-8ae9-d3a55303a0e6">

I have clicked the "Add anyway" button, but couldn't access the `SimpleNetworkService` class.

Is the installation guide up-to-date?

Reply:
added `To Test:` block in PR description to describe the installation